runtime/Makefile

```diff
@@ -99,12 +100,13 @@ endif
 .PHONY: check-tidy
 check-tidy: | qir
 	@echo "build Catalyst Runtime with RUNTIME_CLANG_TIDY=ON"
-	rm -rf 
+	rm -rf
```

Comment:
Mmm... this line led me to the following [link](https://unix.stackexchange.com/questions/553731/why-does-rm-manual-say-that-we-can-run-it-without-any-argument-when-this-is-not). But I think we don't need this line?

Reply:
```suggestion
```